pages/resources/index.vue

```diff
@@ -197,15 +182,30 @@ export default {
      */
     fetchResults: function() {
       this.isLoadingResources = true
+      let entries = {
+        content_type: this.$route.query.type,
+        limit: this.resourceData.limit,
+        skip: this.resourceData.skip,
+        include: 2
+      }
 
-      client
-        .getEntries({
-          content_type: this.$route.query.type,
+      if (
+        this.$route.query.type === 'Platform' ||
+        this.$route.query.type === 'Tool'
+      ) {
+        entries = {
```

Comment:
nit: Another way to do this would be to `Object.assign()` the `entries` object with the new keys.